CIP-0071/README.md

```diff
@@ -0,0 +1,246 @@
+
+## Potential Disadvantages
+
+There are several potential disadvantages to this proposal that may be avoided by the use of a native token or other voting mechanism.  We enumerate some here explicitly so projects can understand where this protocol may or may not be appropriate to use:
+
+- Projects concerned with token proliferation and confusing their user base with the creation of multiple new assets might want to avoid this standard as it requires one new token policy per vote/initiative
+- Projects wishing to create a "secret ballot" that will not be revealed until after polls close should use this because the datum votes appear on-chain (and typically inline)
+  - Performing an encrypted vote on-chain with verifiable post-vote results is an exercise left to the standard's implementer
+- Projects wishing for anonymity in their votes should not use this standard as each vote can be traced to a reference asset
+
+## Optional Recommendations
+
+In no particular order, we recommend the following implementation details that do not impact the protocol, but may impact user experience:
+
+- The mapping function described in the [Plutus minting policy for ballots](#Ballot---Plutus-Minting-Policy) should likely be some sort of prefixing or suffixing (e.g., "Ballot #1 - <REFERENCE NFT>"), and NOT the identity function.  Although the asset will be different than the reference NFT due to its differing policy ID, users are likely to be confused when viewing these assets in a token viewer.
+- The "ballot" NFT should have some sort of unique metadata (if using [CIP-25](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0025)), datum (if using [CIP-68](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0068)) or other identification so that the users can engage with the ballot in a fun, exciting way and to ensure there is no confusion with the reference NFT.
+- The "vote" represented by a datum will be easier to debug and analyze in real-time if it uses the new "inline datum" feature from Vasil, but the protocol will still work on Alonzo era transactions.
+- The "ballot box" smart contract should likely enforce that the datum's "voter" field is respected when returning the ballots to users after voting has ended to provide greater transparency and trust for project participants.
```

Comment:
One thing I am missing from the justifications here is why this solution addresses the problem stated in the motivation. As far as I understood, one of the core motivation was the cost of minting governance tokens (min ada value), but this proposal also proposes to mint ballot NFT and only, _optionally_ return the ballot NFT to the voter. So arguably, the proposal only _optionally_ addresses the motivation. Since this is stated as a motivation, I think the proposal should better describes how it solves this issue systematically.

Reply:
That's fair.  I think we can then enforce that the ballot has to be either:

a) Returned to the user
b) Burned and the ADA unlocked returned to the user

Thoughts?

Reply:
I'd have a slight preference for "Burned and ADA returned to the user" because most users wouldn't probably know how to do that in the first place. Plus, it makes the protocol complete this way by cleaning up after oneself.